src/middleware/github-server-app-middleware.ts

```diff
@@ -56,9 +56,9 @@ export const GithubServerAppMiddleware = async (req: Request, res: Response, nex
 			uuid: undefined, //undefined for cloud
 			hostname: GITHUB_CLOUD_BASEURL,
 			clientId: envVars.GITHUB_CLIENT_ID,
-			gitHubClientSecret: envVars.GITHUB_CLIENT_SECRET,
-			webhookSecret: envVars.WEBHOOK_SECRET,
-			privateKey: await keyLocator(undefined)
+			getDecryptedGitHubClientSecret: async () => envVars.GITHUB_CLIENT_SECRET,
```

Comment:
Shouldn't it be `Promise.resolve(...)`? Otherwise, in one case we are returning a promise, while in the other we return a plain value... or not?

Reply:
js engine will wrap it in Promise anyway (due to the `async` keyword)